deployEthereum.js

```diff
@@ -0,0 +1,115 @@
+const fs = require('fs');
```

Comment:
Can we put this in the scripts folder along with the other `deploy....js` scripts?

Reply:
I also think that this isn't necessary, as this functionality should all be taken care of by the `deployDAOs.js` and `deployDAO.js` scripts

Reply:
oh okay, apologies, I'm now seeing that there's a custom common DAO factory and what not... maybe just take all these common specific scripts and put them in a sub folder in `scripts/` called `common/`?